dnsapi/dns_lua.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
+
```

Comment:
Please add your github address here,  in case there is a bug found, the user can know where to report bug.